bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/BookKeeperBuilder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017 The Apache Software Foundation.
```

Comment:
This is wrong. In ASF, we don't need this. it is only used by projects outside of ASF.